core/domain/entities/routing/handlers/admin/EspressoEventEditor.php

```diff
@@ -40,9 +40,8 @@ public function matchesCurrentRequest(): bool
      */
     protected function registerDependencies()
     {
-        $this->dependency_map->registerDependencies(
-            'EventEspresso\core\domain\services\admin\events\editor\EventEditorGraphQLData',
-            [
+        $editor_dependencies = [
+            'EventEspresso\core\domain\services\admin\events\editor\EventEditorGraphQLData' =>[
                 'EventEspresso\core\domain\entities\admin\GraphQLData\Datetimes'                 => EE_Dependency_Map::load_from_cache,
```

Comment:
instead of calling `$this->dependency_map->registerDependencies()` repeatedly, 
i put everything in an array and then loop over it at the bottom of this method